tests/tasks/setup/kit-cluster/default.yaml

```diff
@@ -81,25 +84,62 @@ spec:
               - name: scheduler
                 args:
                   - --kube-api-qps=$(params.scheduler-qps)
+        etcd:
+          spec:
+            nodeSelector: 
+              node.kubernetes.io/instance-type: $(params.cp-instance-type)
+            containers: 
+            - name: etcd
+              args:
+                - --quota-backend-bytes=8589934592
```

Comment:
KIT-operator already has this configuration, so we don't need to pass this flag

Reply:
You mean that's the default value in `KIT operator` ? Or is it only hardcoded to just that value ?